jiwonjang/routes/postRouter.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+
+const postController = require("../controllers/postController");
+
+const router = express.Router();
+
+router.post("", postController.createPost);
+router.patch("", postController.changeModifyPost);
+router.delete("/:postId", postController.changeDeletePost);
```

Comment:
함수 이름에 동사를 두 번씩 안 사용해도 될 것 같아요. 아래와 같이 수정하는 것이 좋아 보입니다.
```suggestion
router.patch("", postController.modifyPost);
router.delete("/:postId", postController.deletePost);
```